va/validation-authority_test.go

```diff
@@ -169,6 +171,7 @@ func brokenTLSSrv(t *testing.T, stopChan, waitChan chan bool) {
 func TestSimpleHttp(t *testing.T) {
 	va := NewValidationAuthorityImpl(true)
 	va.DNSResolver = core.NewDNSResolver(time.Second*5, []string{"8.8.8.8:53"})
+	va.Stats, _ = statsd.NewNoopClient()
```

Comment:
I know this isn't your code, but please change all of these 8.8.8.8:53s to some constant.